src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background-color: $white-color;
+
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+   &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 37px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    margin-top: 0;
+
+    color: $main-text-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    margin-top: 0;
+
+    color: $second-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews-info {
```

use nesting here & check all your code
```suggestion

    &-info {
```